src/main/scala/com/rbmhtechnology/calliope/serializer/SequencedEventSerializer.scala

```diff
@@ -61,10 +64,14 @@ abstract class SequencedEventSerializer(system: ExtendedActorSystem, payloadSeri
       .setCreationTimestamp(sequencedEvent.creationTimestamp.toEpochMilli)
 
   def sequencedEvent(format: SequencedEventFormat): SequencedEvent[_] =
-    SequencedEvent(
-      payloadSerializer.payload(format.getPayload),
-      format.getSourceId,
-      format.getSequenceNr,
-      Instant.ofEpochMilli(format.getCreationTimestamp)
-    )
+    try {
+      SequencedEvent(
+        payloadSerializer.payload(format.getPayload),
+        format.getSourceId,
+        format.getSequenceNr,
+        Instant.ofEpochMilli(format.getCreationTimestamp)
+      )
+    } catch {
+      case NonFatal(err) => throw new PayloadNotSerializableException("Unable to deserialize payload", err, format.getPayload)
```

Comment:
I think this is what this implementation does. It reports the first encountered error correctly and that is the `PayloadFormat` deserialization fails inside the `SequencedEvent`. 

I deemed it would be important to include the correct serializeID/manifest in the `PayloadNotSerializableException` so it is actually meaningful. Otherwise there will only be the id/manifest of the `SequencedEventSerializer` in the exception no matter what actual payload was in the `SequencedEvent`.